pysatNASA/instruments/timed_guvi.py

```diff
@@ -59,20 +59,22 @@
 
 platform = 'timed'
 name = 'guvi'
-tags = {'': ''}
+tags = {'l1c_img': 'Level 1C imaging disk high resolution data',
+        'l1c2_img': 'Level 1C imaging disk low resolution data',
+        'l1c_spect': 'Level 1C spectrograh disk high resolution data',
+        'l1c2_spect': 'Level 1C spectrograph disk low resolution data'}
 inst_ids = {'': [tag for tag in tags.keys()]}
 
 pandas_format = False
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {'': {'': dt.datetime(2005, 6, 20)}}
+_test_dates = {'': {tag: dt.datetime(2005, 6, 28) for tag in tags}}
```

Comment:
```suggestion
_test_dates = {'': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()}}
```

This is just a pysat stylistic preference that also should be changed a few other places in this file.